### PR TITLE
Programming books

### PR DESCRIPTION
**Pull request by KonstantinaStrantzali**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo
Add recource
### Description
HTML & CSS Greek manual

### Why is this valuable (or not)?
Readers can learn HTML & CSS fundamentals

### How do we know it's really free?
It's published online without any licence written

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-el.md [hunk 1]
```diff
@@ -23,6 +23,7 @@
 ### <a id="html&css"></a> ΗTML & CSS 
 
 * [Eισαγωγή στην HTML & CSS](https://www.ebooks4greeks.gr/eisagwgh-sthn-html-css) (PDF) - Ανδρέας Κομνηνός
```
Comment (Abinashbunty) [inline books/free-programming-books-el.md]: It should be: 
* [Eισαγωγή στην HTML & CSS](https://www.ebooks4greeks.gr/eisagwgh-sthn-html-css) - Ανδρέας Κομνηνός (PDF)

File: books/free-programming-books-el.md [hunk 2]
```diff
@@ -23,6 +23,7 @@
...
+* [Eισαγωγή στην HTML](https://www.ebooks4greeks.gr/eisagwgh-sthn-html) (PDF) - ΚΑΛΛΙΠΟΣ
```
Comment (Abinashbunty) [inline books/free-programming-books-el.md]: Similarly, this should be:
* [Eισαγωγή στην HTML](https://www.ebooks4greeks.gr/eisagwgh-sthn-html) - ΚΑΛΛΙΠΟΣ (PDF)

**Review by Abinashbunty — CHANGES_REQUESTED** [review]

**Comment by Abinashbunty** [comment]
By the time I was suggesting the change, you already implemented it. 😄

**Review by Abinashbunty — APPROVED** [review]

File: books/free-programming-books-el.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 * [C](#c)
 * [C++](#cpp)
+* [ΗTML & CSS](#html&css)
```
Comment (eshellman) [inline books/free-programming-books-el.md:5]: use 'and' not '&'

File: books/free-programming-books-el.md [hunk 4]
```diff
@@ -31,7 +38,7 @@
 
 ### JavaScript
 
-* [HTML5-JavaScript (Δημιουργώντας παιχνίδια – Ο εύκολος τρόπος)](https://www.ebooks4greeks.gr/html5-javascript)
+* [HTML5-JavaScript (Δημιουργώντας παιχνίδια – Ο εύκολος τρόπος)](https://www.ebooks4greeks.gr/html5-javascript) - courses.e-ce.uth.gr
```
Comment (eshellman) [inline books/free-programming-books-el.md:41]: I don't think courses.e-ce.uth.gr is the author

**Review by eshellman — CHANGES_REQUESTED** [review]

**Pull request closed without merging**